.scrutinizer.yml

```diff
@@ -16,5 +16,17 @@ filter:
     excluded_paths:
         - 'vendor/*'
         - 'web/skins/*'
-        - 'skins/10h16/web/_js/lib/*'
+        - 'skins/10h16/*'
         - 'node_modules/*'
+
+build:
+    nodes:
+        analysis:
+            tests:
+                override:
+                - php-scrutinizer-run
+    tests:
+        override:
+        - true
```

Comment:
What does this change do?

Reply:
It disables unit testing on Scrutinizer, since we're doing that on Travis which then sends the code coverage results to Scrutinizer, no need to have them run twice.